src/app/app.component.ts

```diff
@@ -1,10 +1,87 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, ViewChild, AfterViewInit } from '@angular/core';
+import { Location, LocationStrategy, PathLocationStrategy, PopStateEvent } from '@angular/common';
+import 'rxjs/add/operator/filter';
+import { Router, NavigationEnd, NavigationStart } from '@angular/router';
+import { Subscription } from 'rxjs/Subscription';
+import PerfectScrollbar from 'perfect-scrollbar';
+
+declare const $: any;
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss']
 })
-export class AppComponent {
```

Comment:
why it's appears as your changes, is it not the Eran ' s task that he presented us yesterday?

Reply:
I joined his branch